### ISSUE TITLE
Charting: overflow link cannot be accessed by keyboard tab

### ISSUE THREAD

**Issue by LucyLiYY** [microsoft/fluentui#24438]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Edge
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/LucyLi123/pen/gOeqXMM?editors=0010

## Actual Behavior
The overflow link cannot be accessed by keyboard tab.
Issue repro here: https://codepen.io/LucyLi123/pen/gOeqXMM?editors=0010

## Expected Behavior
fill this out

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
